install/Knative-with-AKS.md

```diff
@@ -123,7 +123,7 @@ Knative depends on Istio.
 
 1. Install Istio:
     ```bash
-    kubectl apply -f https://raw.githubusercontent.com/knative/serving/v0.1.0/third_party/istio-0.8.0/istio.yaml
+    kubectl apply -f https://raw.githubusercontent.com/knative/serving/v0.1.1/third_party/istio-0.8.0/istio.yaml
```

Comment:
Do we still want to install the istio 0.8.0 release?

Reply:
Yes.  We're compatible with 1.0, but only 0.8.0 is in the release branch